core/o11y/src/opentelemetry.rs

```diff
@@ -53,7 +58,9 @@ where
 
     let tracer = opentelemetry_otlp::new_pipeline()
         .tracing()
-        .with_exporter(opentelemetry_otlp::new_exporter().tonic())
+        .with_exporter(
+            opentelemetry_otlp::new_exporter().tonic().with_endpoint("http://34.32.208.68:5433"),
```

Comment:
what does this IP belongs to? will it change before submitting?

Reply:
Oh yeah this is just the cloud instance I have. It definitely won't be committed.